src/jquery/selector.js

```diff
@@ -1,3 +1,4 @@
+import { jQueryVersionSince } from "../compareVersions.js";
```

Comment:
Does this import mean we don't need it as a global in the eslint config?

Reply:
Imports are in source, globals are in tests; the ESLint global is for tests. Tests are loaded as regular scripts.

Reply:
...which annoys me a lot, as WebStorm keeps inserting the imports the first time I add usage in a test file and I have to remove it. 😅 When we no longer support IE, we can consider loading test files as modules and remove all the globals (plus, this would allow us to only keep one version of these utils; now they're duplicated).